best-practices.js

```diff
@@ -51,7 +49,7 @@ module.exports = {
     // disallow usage of __iterator__ property
     'no-iterator': 2,
     // disallow use of labeled statements
-    'no-labels': 2,
+    'no-labels': [2, { 'allowLoop': false, 'allowSwitch': false }],
```

Comment:
`allowLoop` and `allowSwitch` appears to default to `false`. Should be possible to omit them here.